.github/workflows/_build-and-push.yml

```diff
@@ -109,6 +116,17 @@ jobs:
           IMAGE_TAG: ${{ inputs.tag }}
         run: echo "Tag to use - ${IMAGE_TAG}"
 
+      - name: tag and push development container
```

Comment:
name says dev, but the branch check below is main.

Reply:
Yeah that's correct. This is for the annoying development environment that has the no robots setup which we use for external users. I was actually going to ask about environments when I'm back on digideps as we have too many imo and we may be able to ditch some. So the branch is main but the workspace to be used is development but this is the build part so we only need to build this image when it's a merge to main if that makes sense :)

Reply:
Ah, cool. Wasn't clear at first glance, so worth specifying that in the step name. "tag and push user research development container" or something.